uberpoet/buckprojectgen.py

```diff
@@ -136,9 +146,9 @@ def gen_lib_module(self, module_node, loc_per_unit):
         buck_text = self.bzl_lib_template.format(module_node.name, deps, self.wmo_state)
 
         # Make Swift Text
-        file_count = (loc_per_unit * module_node.code_units) / self.swift_file_size_loc
+        file_count = (max(self.swift_file_size_loc, loc_per_unit) * module_node.code_units) / self.swift_file_size_loc
```

Comment:
its now possible that passing a loc file might generate way too little LOC for a module, this way taking the max it is guaranteed we always produce at least 1 file for such modules. Happy to discuss alternatives

Reply:
I think that is reasonable since a file is pretty small